addons/l10n_vn/migrations/14.0.2.0.1/post-migration.py

```diff
@@ -12,7 +12,7 @@ def _fix_revenue_deduction_accounts_code(env):
     vn_template = env.ref('l10n_vn.vn_template')
     for company in env['res.company'].with_context(active_test=False).search([('chart_template_id', '=', vn_template.id)]):
         for incorrect_code, correct_code in FIXED_ACCOUNTS_MAP.items():
-            account = env['account.account'].search([('code', '=', incorrect_code), ('company_id', '=', company.id)])
+            account = env['account.account'].search([('code', '=', incorrect_code), ('company_ids', '=', company.id)])
```

Comment:
No, this script is for the upgrade of Odoo 14.
It should stay `company_id`.
```suggestion
            account = env['account.account'].search([('code', '=', incorrect_code), ('company_id', '=', company.id)])
```